docs/topics/mta-introduction-questionnaire.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+
+:_content-type: REFERENCE
+[id="mta-introduction-questionnaire_{context}"]
+= MTA questionnaire
+
+Migration Toolkit for Applications (MTA) uses questionnaires for assessing the risks involved in containerizing an application. By default, this procedure is required prior to reviewing the application.
+
```

Comment:
Required? What mechanism enforces this? What is meant by "reviewing the application."?